scripts/generate_site_assets.py

```
@@ -265,30 +264,32 @@ def process_entity(args):
+            for category in categories:
+                filename = f"{subdiv or 'ALL'}_{language}_{category}"
+                try:
+                    h_obj = holidays_func(
+                        code,
+                        subdiv=subdiv,
+                        years=range(DEFAULT_YEAR_START, DEFAULT_YEAR_END + 1),
+                        language=language,
+                        categories=category,
+                    )
+                    write_assets(h_obj, filename, out_dir)
+                except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)